### PR TITLE
Update LineCounter readme

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: samples/linecounter/README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ name: LineCounter
 description: Sample that illustrates using blob and event hub clients along with ASP.NET Core integration, distributed tracing and hosted services.
 ---
 
+# LineCounter
+This is a sample app that illustrates using blob and event hub clients along with ASP.NET Core integration, distributed tracing and hosted services.
```
Comment (kaerm) [inline samples/linecounter/README.md]: nit: I guess they keep on insisting on calling it Event Hubs (plural)

File: samples/linecounter/README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ name: LineCounter
...
+
```
Comment (kaerm) [inline samples/linecounter/README.md]: Do you want to add one more sentence on what the app actually does?

File: samples/linecounter/README.md (hunk 1 above)
Comment (JoshLove-msft) [reply samples/linecounter/README.md]: Updated

File: samples/linecounter/README.md (hunk 2 above)
Comment (JoshLove-msft) [reply samples/linecounter/README.md]: Added more detail

**Review by kaerm — APPROVED** [review]
👍

**Pull request merged**